package.json

```diff
@@ -15,28 +17,28 @@
     "link",
     "monorepo"
   ],
-  "author": "Viktor Varland",
+  "author": "Viktor Varland <viktor@dhis2.org>",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "@dhis2/code-style": "^1.7.1",
+    "@dhis2/cli-helpers-engine": "^0.10.1",
     "@vardevs/io": "^1.5.0",
     "@vardevs/log": "^1.0.0",
```

Comment:
I have a simple [reporter](https://github.com/dhis2/cli/blob/master/packages/helpers-engine/lib/reporter.js) utility in the engine currently, but it could probably be expanded and optionally split out into a separate utility.  I didn't go through and replace the logging here yet though.